diff --git a/llama_stack/models/llama/datatypes.py b/llama_stack/models/llama/datatypes.py
index 7f1ebed5..f5e26ec3 100644
--- a/llama_stack/models/llama/datatypes.py
+++ b/llama_stack/models/llama/datatypes.py
@@ -31,7 +31,7 @@ class BuiltinTool(Enum):
 
 
 Primitive = str | int | float | bool | None
-RecursiveType = Primitive | list[Primitive] | dict[str, Primitive]
+type RecursiveType = Primitive | list[RecursiveType] | dict[str, RecursiveType]
 
 
 class ToolCall(BaseModel):
