diff --git a/apps/assisted-disconnected-ui/vite.config.ts b/apps/assisted-disconnected-ui/vite.config.ts
index 62cd7b5a..1057f1be 100644
--- a/apps/assisted-disconnected-ui/vite.config.ts
+++ b/apps/assisted-disconnected-ui/vite.config.ts
@@ -5,7 +5,7 @@ import 'zx/globals';
 
 export const getDefaultValuesForEnvironmentVariables = async () => {
   $.verbose = false;
-  const commitSignature = (await $`git rev-parse --short HEAD`).toString().trim();
+  const commitSignature = "x";
 
   return {
     AIUI_APP_IMAGE_REPO: 'quay.io/edge-infrastructure/assisted-disconnected-ui',
diff --git a/apps/assisted-ui/vite.config.ts b/apps/assisted-ui/vite.config.ts
index 9d034372..69917bb8 100644
--- a/apps/assisted-ui/vite.config.ts
+++ b/apps/assisted-ui/vite.config.ts
@@ -5,7 +5,7 @@ import 'zx/globals';
 
 export const getDefaultValuesForEnvironmentVariables = async () => {
   $.verbose = false;
-  const commitSignature = (await $`git rev-parse --short HEAD`).toString().trim();
+  const commitSignature = "x";
 
   return {
     AIUI_APP_API_ROOT: '/api/assisted-install',
